core/controllers/acl_decorators.py

```diff
@@ -1816,8 +1771,6 @@ def can_user_edit_skill(user, skill_rights):
             bool. Whether the given user can edit the given skill.
         """
 
-        if skill_rights is None:
```

Comment:
Why delete?

Reply:
This function(can_user_edit_skill) is called in test_can_edit_skill which already checks for skill rights to be None [here](https://github.com/oppia/oppia/blob/develop/core/controllers/acl_decorators.py#L1851)

Reply:
Actually having can_user_edit_skill as a separate function is a bit confusing (for reasons like this). Could you get rid of it and inline the logic in test_can_edit_skill, if nothing else is using it? That would make things clearer.

Reply:
Done